CycloneDX/Services/ProjectFileService.cs

```diff
@@ -240,24 +239,19 @@ public async Task<HashSet<DotnetDependency>> GetProjectDotnetDependencysAsync(st
         public async Task<HashSet<DotnetDependency>> RecursivelyGetProjectDotnetDependencysAsync(string projectFilePath, string baseIntermediateOutputPath, bool excludeTestProjects, string framework, string runtime)
         {
             var dotnetDependencys = await GetProjectDotnetDependencysAsync(projectFilePath, baseIntermediateOutputPath, excludeTestProjects, framework, runtime).ConfigureAwait(false);
-            foreach (var item in dotnetDependencys)
-            {
-                item.IsDirectReference = true;
-            }
-
```

Comment:
This was wrong in the first place and only compensated the missing `IsDirectReference` from `PackagesFileService.GetDotnetDependencysAsync()`. So by fixing it there, this is not necessary anymore. I assume this is here from before `--includeProjectReferences` was around, but that handles this already.